docker-compose-staging.yml

```diff
@@ -54,7 +53,8 @@ services:
       - 80:80
       - 443:443
     restart: always
-    depends_on: telescope
+    depends_on:
+      - telescope
```

Comment:
Why are we switching the syntax here? Was it previously not being picked up correctly by docker-compose?

Reply:
Also, if we are switching syntax to an array, should it not be (see spacing of dash):

```yml
    depends_on: 
    -  telescope
```

Reply:
The syntax is all handled by Prettier automatically

@raygervais not sure if you're using VSCode, but if so, make sure you install the recommended workspace extensions/settings to get format-on-save, eslint, etc.  If not, it will happen via our husky hook on commit.

Reply:
@raygervais I tried:
```
depends_on: telescope
```
and

```
depends_on:
- telescope
```

Reply:
Alright, good to know @humphd and @manekenpix. I'll assume prettier knows what they're doing and mark this one resolved.